### DISCUSSION TITLE
自定义主题功能讨论区

### DISCUSSION THREAD

**Discussion by caol64** [caol64/wenyan#9] [General]
自定义主题相关功能讨论，问题答疑。

自定义主题使用教程：[点我](https://babyno.top/posts/2024/11/wenyan-supports-customized-themes/)

所有属性速查表：[点我](https://yuzhi.tech/docs/wenyan/themes)

**Comment by caol64** [comment]
鉴于自定义主题入门门槛较高，虽然出过一些教程，但即使对于专业人士也难免挠头。因此我希望引入一些工具来简化这方面的工作。

第一是添加导入和转换工具。对于成熟社区大量已有的主题可以直接导入，迅速形成自己的主题。

第二是“主题编辑器”增加“开发者模式”和“普通模式”。“开发者模式”将完全开放CSS的编辑功能。“普通模式”将不再要求用户掌握CSS知识，用户可以直接动用鼠标和键盘修改页面元素。

另外会开放内置主题的代码，提供更多参考信息。

**Reply by zw-95** [reply]
建议优化一下自定义主题功能，类似[markdown在线编辑器](https://markdown.com.cn/editor/#)，可以实时修改css、修改需要预览的内容，增加主题的名称、顺序设置等。现在调主题有点困难

**Comment by caol64** [comment]
该主题[#13 ](https://github.com/caol64/wenyan/discussions/13#discussioncomment-11412548)由工具+手工微调生成。目前看来手工微调还是必不可少的。
